ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil

```diff
@@ -0,0 +1,64 @@
+#!nsh
+#
+# @name Standard vtol HITL (gazebo standard_vtol model)
```

Comment:
Shouldn't this be "Standard VTOL HITL" ?

Reply:
Might be better as:
```
@name Standard VTOL HITL (Gazebo only)
```

Reply:
Actually, the existing HIL frames have names like HIL ...., so perhaps this should be named 
"HIL Standard VTOL QuadPlane"